lib/kafka_ex.ex

```diff
@@ -29,10 +31,10 @@ defmodule KafkaEx do
   def create_worker(name, worker_init \\ [])
 
   def create_worker(name, worker_init) do
-    worker_init = case worker_init do
-      [] -> [uris: Application.get_env(:kafka_ex, :brokers)]
-      _   -> worker_init
-    end
+    defaults = [uris: Application.get_env(:kafka_ex, :brokers),
+                consumer_group: Application.get_env(:kafka_ex, :consumer_group)]
+
+    worker_init = Keyword.merge(defaults, worker_init)
```

Comment:
taking a second pass through this, I see that the intent is different from the original implementation, but based on explanation elsewhere this will break support for kafka < 0.8.2 also, there is an implicit assumption that you can pass a completely different set of kafka brokers when creating a worker that this will also break